salt/modules/rabbitmq.py

```diff
@@ -473,7 +473,7 @@ def check_password(name, password, runas=None):
             return False
         return True
 
-    cmd = ('rabbit_auth_backend_internal:check_user_login'
+    cmd = ('rabbit_auth_backend_internal:user_login_authentication'
```

Comment:
do the same thing what puppet folks: https://github.com/puppetlabs/puppetlabs-rabbitmq/pull/322/files#diff-c62722635df69c579feeb9bf9534b82dR48

"version" is available in the code, so it's not a problem to keep backward compatibility.